content/concepts/consensus-network/amendments/amendments.md

```diff
@@ -223,6 +225,15 @@ TrustSetAuth
 ```
 
 
+## Retiring Amendments
```

Comment:
This title makes me think that we're talking about how to make amendments stop working, but it seems to be actually referring to "forgetting" code from before a certain amendment. Maybe something like "Retiring Old Behavior" would be more accurate, but it's also not that intuitive...